### ISSUE TITLE
[Adaptive Cards - Elements Properties] - Screen reader is not announcing complete information while navigating on the option present under the more option.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#9036]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: Dev (OS Build 27695.1000)
Browser Version: Version 130.0.2849.1 (Official build) dev (64-bit)
Screen Reader: Narrator

**Repro Steps:**
1. Open the above URL on the Edge Dev browser.
2. Press tab key and navigate to the Designer tab item and press enter key to select it.
3. Press tab key navigate to the editor section and select any field.
4. Press tab key and navigate to the 'Element Property' button and press enter key to activate it.
5. Navigate to more option button using tab key and activate it using enter key.
6. Navigate the list item and observe the issue.

**Actual Result:**
While navigating on the list items present under more option button, screen reader is not conveying the complete information about the option. It is just announcing as **'$root 1 of 1 level 1 expanded current selected.'**

**Expected Result:**
Screen Reader should convey the complete information for the list items. It should announce as **'$root object 1 of 1 level 1 expanded current selected.'**

**Note:**
Same issue is repro throughout the element properties screen.
Same issue is repro with NVDA.

**User Impact:**
It will impact the screen reader user as they will not get the complete information about the options due to this there might be a chance that important information miss by them.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

Have feedback to share on Bugs? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots
![While navigating on the option present under combo box, screen reader is not conveying the complete information  It is only announcing as $root](https://github.com/user-attachments/assets/7e8d1070-4941-4813-a9a1-b561630f9321)

https://github.com/user-attachments/assets/0f9876e6-a29a-4e13-af6f-6ae67f2c7918

### Card JSON

```JSON
NA
```
